src/contrib/cluster/Akka.Cluster.Sharding/reference.conf

```diff
@@ -268,6 +268,20 @@ akka.cluster.sharding {
 }
 # //#sharding-ext-config
 
+akka.cluster.sharded-daemon-process {
+  # Settings for the sharded dameon process internal usage of sharding are using the akka.cluste.sharding defaults.
+  # Some of the settings can be overriden specifically for the sharded daemon process here. For example can the
+  # `role` setting limit what nodes the daemon processes and the keep alive pingers will run on.
+  # Some settings can not be changed (remember-entitites and related settings, passivation, number-of-shards),
+  # overriding those settings will be ignored.
+  sharding = ${akka.cluster.sharding}
```

Comment:
This seems to work, but `akka.cluster.sharding.coordinator-singleton = ${akka.cluster.singleton}` does not. I guess the current hocon implementation cannot load a config section from another file!?

Reply:
I don't think so - cc @Arkatufus